src/plugins/management/public/types.ts

```diff
@@ -101,6 +100,6 @@ export interface AppDependencies {
 
 export interface ConfigSchema {
   deeplinks: {
-    navLinkStatus: keyof typeof AppNavLinkStatus;
+    navLinkStatus: 'default' | 'visible';
```

Comment:
I left this navLinkStatus config to not break Cloud deployment using it. It could be deprecated in a following work as I don't think it is being used anymore.